### PR TITLE
Metadata extraction

### PR DESCRIPTION
**Pull request by geramirez**
``` bash
$ nosetests --with-coverage
........
Name                        Stmts   Miss  Cover   Missing
---------------------------------------------------------
textextraction                  0      0   100%   
textextraction.extractors      67      0   100%   
---------------------------------------------------------
TOTAL                          67      0   100%   
----------------------------------------------------------------------
Ran 8 tests in 115.888s

```

### CODE REVIEW

**Comment by khandelwal** [comment]
Why is the celery stuff included here? Instead of including that here, we should keep this code base simple and just mention that celery would be one way of using the doc_processing_toolkit.

**Comment by khandelwal** [comment]
By adding in celery, we're simply adding in too many dependencies.

File: README.md [hunk 1]
```diff
@@ -9,23 +9,35 @@ Python library to extract text from any file type compatiable with [TIKA](http:/
...
 1. Download tika-app-1.7.jar from [Apache Tika](http://tika.apache.org/)
 2. Mac: `brew install ghostscripts` Ubuntu: `sudo apt-get install ghostscript`
-3. `brew install tesseract` Ubuntu: `sudo apt-get install tesseract-ocr`
+3. Mac: `brew install tesseract` Ubuntu: `sudo apt-get install tesseract-ocr`
+4. Mac: `brew tap homebrew/x11` and `brew install xpdf` Ubuntu: `sudo apt-get install poppler-utils`
 
 ##### Usage
-These script assume that Apache Tika server is running in text mode.
-Start Tika Server
+These script assume that two instances of Tika server are running, one to extract meta data and the other to extract text.
```
Comment (khandelwal) [inline README.md]: What? This doesn't make sense. Leave any notions of parallelization out of this library I think.

Comment (geramirez) [inline README.md]: Ok, I've changed everything to go through one instance of Tika.

File: build/lib/textextraction/extractors.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+class TextExtraction:
+    """ The TextExtraction class contains functions for extracting and saving
+    metadata and text from all files compatible with Apache Tika"""
+
+    def __init__(self, doc_path, tika_port=9998):
+
+        self.doc_path = doc_path
+        self.root, self.extension = os.path.splitext(doc_path)
+        self.tika_port = tika_port
+        self.text_arg_str = 'curl -T {0} http://localhost:{1}/tika' + \
```
Comment (khandelwal) [inline build/lib/textextraction/extractors.py]: Should specify the URL for the tika servers somewhere else? As a class variable perhaps? It seems weird to have that hard-coded here.

Comment (geramirez) [inline build/lib/textextraction/extractors.py]: Ok, I changed this.

**Pull request merged**